genesis-blocks/ganache-standalone

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash
+# This script will start a private gth blockchain with two miners and two node.
```

Comment:
I refer to my old comment here.

https://github.com/EYBlockchain/nightfall_3/pull/531#pullrequestreview-897095300

This needs to work with `./start-nightfall`. 
So we don't need these new scripts 
- `genesis-blocks/ganache-standalone`
- `genesis-blocks/genesis-down`
- `genesis-blocks/genesis-nightfall`